WordPress/src/main/res/layout/reader_blog_section_view_new.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/blog_section_image_blog_avatar"
+        style="@style/ReaderImageView.Avatar"
+        android:layout_marginEnd="@dimen/margin_medium"
+        android:background="@drawable/bg_oval_placeholder"
+        android:importantForAccessibility="no"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/blog_section_text_blog_name"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        tools:ignore="RtlSymmetry" />
+
+    <ViewStub
```

Comment:
❓ I might've missed something, but I couldn't find this `ViewStub` being used.

Reply:
I just used it the same way as it's being used in the original layout but we don't really need to do it this way, will fix it and remove the `ViewStub`.